### DISCUSSION TITLE
Welcome to StarterHive Discussions!

### DISCUSSION THREAD

**Discussion by ArslanYM** [ArslanYM/StarterHive#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions related to the issue youre working on.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VarunJoshi10** [comment]
Hello Folks I am Varun and I am here to learn more and more about how to contribute to open-source project.
I am currently learning full stack and devops. 
Thank you @ArslanYM I have started to contribute to open-source and will be very happy to provide value to this Community. 
You can Connect with me on :
- [Twitter](https://twitter.com/_Varun_08?t=Dx7bk67gNjYJWPasVzfwuA&s=09)

**Reply by ArslanYM** [reply]
@VarunJoshi10 Welcome to open source.
